lib/ui/widgets/schedule_settings.dart

```diff
@@ -126,8 +126,9 @@ class _ScheduleSettingsState extends State<ScheduleSettings> {
       tiles.add(Padding(
         padding: const EdgeInsets.fromLTRB(15.0, 8.0, 15.0, 8.0),
         child: Text(
-          model.scheduleActivitiesByCourse[courseActivitiesAcronym].first
-              .courseTitle,
+          '${model.scheduleActivitiesByCourse[courseActivitiesAcronym].first
+              .courseTitle} (${model.scheduleActivitiesByCourse[courseActivitiesAcronym].first
+              .courseAcronym})',
```

Comment:
```suggestion
          '${model.scheduleActivitiesByCourse[courseActivitiesAcronym].first
              .courseAcronym} - ${model.scheduleActivitiesByCourse[courseActivitiesAcronym].first
              .courseTitle}',
```

The course acronym is always put before the course title if you check SignÉTS, PlanÉTS, MonÉTS, etc. 😄